tests/generation/SettingsTest.py

```diff
@@ -16,6 +17,13 @@
 
 class SettingsTest(unittest.TestCase):
 
+    def path_leaf(self, path):
+        """
+        The function returns file name from a path
```

Comment:
Not necessary, but it's better to write it this way (just for the sake of consistency in our codebase)
```py
"""
:return: The file name from the given path.
"""
```